Cargo.toml

```diff
@@ -54,16 +57,15 @@ minidom_writer = "1"
 num-traits = "0.2"
 pretty_assertions = "0.6"
 proj = { version = "0.14", optional = true }
-quick-xml = "0.17"
+quick-xml = "0.18"
```

Comment:
It feels to me that except for `quick-xml`, all the other should already be to the last version with a simple `cargo update` (see [Specifying Dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements)).

Reply:
OK, so the way I see it : we just change the toml so that the behavior is explicit, like you suggested before (use only major versions above 1.x.x, minor for 0.x.x)